### PR TITLE
fix(docker): push to latest on tagging a new version

### PR DESCRIPTION
**Pull request by lamroger**
We want latest to match the latest tag.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM python:3.8
 
 RUN pip install --no-cache-dir --upgrade pip && \
-    pip install --no-cache-dir gradient
```
Comment (bbatha) [inline Dockerfile:4]: This definitely shouldn't pin. You could add an `ENV VERSION` and set `-e VERSION="$CIRCLECI_TAG` to produce a container with the exactly built version.

**Review by bbatha — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (lamroger) [reply Dockerfile:4]: Gotcha - how does the update look? i removed some pushes and only push on tag two versions

Comment (lamroger) [reply Dockerfile:4]: i used arg instead of var

Comment (bbatha) [reply Dockerfile:4]: arg's good

**Review by bbatha — APPROVED** [review]

**Pull request merged**